mmedit/models/components/layers/mean_shift.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class MeanShift(nn.Conv2d):
```

Comment:
MeanShift is a classical alogrithm, so I do not like this name. Do you have alternative choices? Like `FeatureShift` or so?